lib/strategy.js

```diff
@@ -45,12 +46,13 @@ function Strategy(options, verify) {
     options = {};
   }
   if (!verify) { throw new TypeError('LocalStrategy requires a verify callback'); }
-  
+
   this._usernameField = options.usernameField || 'username';
   this._passwordField = options.passwordField || 'password';
-  
+  this._totpField = options.totpField || 'totp';
+
   passport.Strategy.call(this);
-  this.name = 'local';
+  this.name = 'local-with-totp';
```

Comment:
The strategy needs a unique name so that it can be [invoked](https://github.com/artsy/artsy-passport/compare/master...anandaroop:totp-strategy-support#diff-c7d8f6e9781b93e5f26d9a3fbfbf13c0R20) in the consuming app